runtime/lua/vim/lsp/buf.lua

```diff
@@ -202,6 +202,65 @@ function M.outgoing_calls()
   end)
 end
 
+--- List workspace folders.
+function M.list_workspace_folders()
```

Comment:
Should return a list of the folders, users can print if they want.

Reply:
Added